static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -270,9 +270,8 @@ function GroupReplaysTable({
       <ReplayCountHeader>
         <IconUser size="sm" />
         {t(
-          'Replay captured %s experiencing this issue across %s.',
-          tn('%s user', '%s users', replayCount ?? 0),
-          tn('%s event', '%s events', group.count)
```

Comment:
why did we get rid of this line?

Reply:
I wasn't sure how to word it properly with the events count, updated to bring it back, lmk if you have any suggestions

Reply:
with @jas-kas 's suggestion i think it sounds fine!